src/log.hpp

```diff
@@ -62,14 +62,21 @@
 
 namespace lg {
 
-// Prefix and extension for log files. This is used both to generate the unique
-// log file name during startup and to find old files to delete.
+// Prefix and extension for log files.
+// This is used to find old files to delete.
 const std::string log_file_prefix = "wesnoth-";
 const std::string log_file_suffix = ".log";
+// stdout file for Windows
+const std::string out_log_file_suffix = ".out.log";
 
 // Maximum number of older log files to keep intact. Other files are deleted.
 // Note that this count does not include the current log file!
+// double for Windows due to the separate .log and .out.log files
+#ifdef _WIN32
+const unsigned max_logs = 16;
+#else
 const unsigned max_logs = 8;
+#endif
```

Comment:
I think it might be slightly better if this were arranged in a way that doesn't require editing two places. For example, like below. There may be even better possibilities as well.
```suggestion
const unsigned max_logs = 8
// double for Windows due to the separate .log and .out.log files
#ifdef _WIN32
  * 2;
#endif
```

Reply:
Wouldn't that fail to compile due to missing the semi-colon on non-Windows?